apps/codeforafrica/src/lib/data/common/index.js

```diff
@@ -160,7 +160,7 @@ export async function getPageProps(api, context) {
       return null;
     }
   }
-  const blocks = await blockify(page?.blocks, api, context);
+  const blocks = (await blockify(page?.blocks, api, context)) ?? null;
```

Comment:
What is happening here?

Reply:
Just a check that `page.blocks` is never `undefined`